tools/ci/testlist/arm-02.dat

```diff
@@ -1,4 +1,5 @@
 /arm/[i-k]*,CONFIG_ARM_TOOLCHAIN_GNU_EABI
+-sabre-6quad:libcxx
```

Comment:
This is removed from the CI because?...

Reply:
Yes, here is some finding.
since linker can't find basic_string::~basic_string() oddly:
```
arm-none-eabi-ld: /home/xiaoxiang/backup/os/nuttx/nuttx/staging/libxx.a(libcxx/src/locale.o): in function `__tcf_0':
/home/xiaoxiang/backup/os/nuttx/nuttx/libs/libxx/libcxx/src/locale.cpp:4941: undefined reference to `std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >::~basic_string()'
````
another fix is removing the follow code from _LIBCPP_STRING_V1_EXTERN_TEMPLATE_LIST and 
_LIBCPP_STRING_UNSTABLE_EXTERN_TEMPLATE_LIST:
_Func(_LIBCPP_FUNC_VIS basic_string<_CharType>::~basic_string()) \
this error just happen on Cortex-A9, other arch(Cortex-M4, sim, xtensa and riscv) are fine.

Reply:
So is this a compiler issue?

Reply:
@xiaoxiang781216 will you submit a bug to gcc bugzilla on this?

Reply:
Not yet.

Reply:
@xiaoxiang781216 i have the linker issue that you mentioned above here: https://github.com/apache/nuttx/issues/11227